.github/workflows/run-script.yml

```diff
@@ -9,20 +9,19 @@ jobs:
   run_script_job:
     runs-on: ubuntu-latest
     steps:
-      - name: Check out the code
-        uses: actions/checkout@v3
+      # TODO: consider replacing python/pip/update-web-metadata installs with docker image
       - name: Setup Python
         uses: actions/setup-python@v4
       - name: Upgrade pip
         run: |
           # install pip=>20.1 to use "pip cache dir"
           python -m pip install --upgrade pip wheel
-      - name: Install deps
-        run: python -m pip install -r requirements.txt
       - name: Install package
-        run: python -m pip install -e .
+        run: python -m pip install git+https://github.com/msarahan/update-web-metadata.git@console-scripts
```

Comment:
@msarahan Do you plan to merge with this pointing to your repo or are you just leaving for now for testing?

Reply:
oh i do have a similar question as @willingc , Mike. Many thanks.

Reply:
The plan is to revert the commit that starts with 'REVERT ME' before merging. I'm on the road away from a real computer right now. I might be able to handle that this evening. Please feel free to push to my fork/branch if you want to merge this sooner.